src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonSynchronisationService.kt

```diff
@@ -53,11 +54,10 @@ class ContactPersonSynchronisationService(
   suspend fun personRestrictionUpserted(event: PersonRestrictionEvent) {
     val telemetry =
       mutableMapOf<String, Any>("nomisPersonId" to event.personId, "dpsContactId" to event.personId, "nomisPersonRestrictionId" to event.visitorRestrictionId)
-    val telemetrySuffix = "contactperson-person-restriction-synchronisation"
```

Comment:
dropped this idea of this silly misnamed variable

Reply:
phew